src/interface.jl

```diff
@@ -439,40 +431,54 @@ function build_bnb_callback(
     end
 end
 
-function postsolve(tree, result, time_ref, verbose=false)
+function postsolve(tree, result, time_ref, verbose=false, use_postsolve = true)
```

Comment:
```suggestion
function postsolve(tree, result, time_ref, verbose=false, use_postsolve = true)
```
the last two should be keyword arguments more than default arguments